integration/spring-data/base/src/main/java/com/blazebit/persistence/spring/data/base/repository/AbstractEntityViewAwareRepository.java

```diff
@@ -71,6 +71,7 @@
  * @author Christian Beikov
  * @since 1.2.0
  */
+@Transactional(readOnly = true)
```

Comment:
Is this sensible for updatable EV's?

Reply:
Yeah, the write operations have a normal transactional annotation